core/src/main/resources/templates/execTopics.html

```diff
@@ -295,11 +295,22 @@ <h4 class="mb-3">Shortcuts</h4>
 
 						</tr>
 						<tr>
-							<td width="100%">
+							<td width="100%" ng-show="dashboardDetails.canSwitchTeams == 'false'">
 								<li class="nav-item d-none d-md-block search-box">
```

Comment:
Is there anyway we could write this piece of html (and the previous JS once and import it or inherit it in the other files?

Reply:
I had this issue right from the beginning. Because of angular controllers, its hard to include multiple js/controllers in html files. But will explore again.